snippets/php/outgoing-payment/outgoing-payment-create.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+ //@! start chunk 1 | title=Create outgoing payment
+$newOutgoingPayment = $opClient->outgoingPayment()->create(
+    [
+        'url' => $wallet->resourceServer,
+        'access_token' => $OUTGOING_PAYMENT_GRANT_ACCESS_TOKEN
+    ],
+    [
+        'walletAddress' => $config->getWalletAddressUrl(),
+        'quoteId' => $QUOTE_URL,
+        'metadata' => [
+            'description' => 'Test outgoing payment',
+            'reference' => '1234567890',
+            'invoiceId' => '1234567890',
+            'customData' => [
+                'key1' => 'value1',
+                'key2' => 'value2'
+            ]
+        ],
+    ]
+);
+//@! end chunk 1
+
+//@! start chunk 2 | title=Create outgoing payment with amount
+$newOutgoingPayment = $opClient->outgoingPayment()->create(
+    [
+        'url' => $wallet->resourceServer,
+        'access_token' => $OUTGOING_PAYMENT_GRANT_ACCESS_TOKEN
+    ],
+    [
+        'walletAddress' => $config->getWalletAddressUrl(),
+        'incomingPayment' => $INCOMING_PAYMENT_URL,
+        'debitAmount' => [
+            'value' => '9',
+            'assetCode' => 'USD',
+            'assetScale' => 2
+        ],
+        'metadata' => [
+            'description' => 'Test outgoing payment',
+            'reference' => '1234567890',
+            'invoiceId' => '1234567890',
+            'customData' => [
+                'key1' => 'value1',
+                'key2' => 'value2'
+            ]
+        ],
+    ]
+);
+//@! end chunk 2
```

Comment:
We can just keep the first snippet (the different types of requests should be covered in the guides)